README.md

```diff
@@ -2,69 +2,100 @@
 
 Lightweight Javascript utility for using Artsy's Gemini service to upload directly to S3. Used internally at Artsy, so not useful to general public—but open source by default!
 
-## Example
+## Installation
 
-Add to your script tags
+Use `gemup` one of three ways:
 
-````html
+### Install with Yarn/NPM
+
+```
+yarn add @artsy/gemup
+```
+
+### Add to your script tags
+
+```html
 <html>
   <body>
-    <script src='gemup.js'></script>
+    <script src="gemup.js"></script>
   </body>
 </html>
-````
+```
+
+### Require via browserify
+
+```js
+var gemup = require("gemup");
+```
+
+## Example usage
 
-or require via browserify
+### JSX
```

Comment:
I think we can drop this JSX header and switch the `jsx` language tag below to `javascript`. The code examples below are JavaScript, which I think is expected.

Reply:
done!